src/Configuration/test/ConfigServer.Test/Info.cs

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the Apache 2.0 License.
 // See the LICENSE file in the project root for more information.
 
-namespace Steeltoe.Extensions.Configuration.ConfigServer.ITest;
+namespace Steeltoe.Extensions.Configuration.ConfigServer.Test;
```

Comment:
Changing the name of this project seems likely to cause unresolvable conflict with #973. I'd rather spell out `IntegrationTest` than drop the `I` completely

Reply:
I assumed the `I` was a typo, didn't realize it was referring to integration tests. Renaming to *.IntegrationTest violates the documented naming convention:

> The unit tests for a package should be put in a project with the same name and append .Test to the name.

Alternatively we could rename to *.Integration.Test. Which do you prefer?

Reply:
Let's go with .Integration.Test